src/lib/holocene/select/multi-select.svelte

```diff
@@ -38,7 +38,12 @@
     } else {
       selectedOptions = [...selectedOptions, option];
     }
-    onChange(selectedOptions);
+
+    if (selectedOptions.length === options.length) {
+      onChange([]);
```

Comment:
Wondering if we want this conditional here or in the `onChange`. Might there be an instance were we want all the options to be passed to the `onChange` when they are all selected?

Reply:
You are absolutely right, I'll move it to the onChange. Good catch